Scripts/tests/test_AddData.py

```diff
@@ -0,0 +1,71 @@
+"""
+`test_adddata.py` perform regression testing of adding-data functionality
+"""
+
+from unittest import mock
+import os, glob, subprocess
+from tempfile import TemporaryDirectory
+import pytest
+import DatabankLib
+
+@pytest.fixture(scope="module")
+def tmpWorkDir():
+    with TemporaryDirectory(prefix="dbtestWD_", dir=os.path.dirname(__file__)) as wdir:
+        print(f"Will use following directory for loadings: {wdir}")
+        yield wdir
+
+@pytest.fixture(scope="module")
+def tmpOutDir():
+    with TemporaryDirectory(prefix="Simulations.", 
+                            dir=os.path.join( os.path.dirname(__file__), "Data")
+                            ) as wdir:
+        print(f"Will use following directory for writting: {wdir}")
+        yield wdir
+
+@pytest.fixture(autouse=True, scope="module")
+def header_module_scope():
```

Comment:
Should this fixture be global and reused by different tests? It looks like there is code duplication between tests, but this can be solved with [config file](https://docs.pytest.org/en/stable/reference/fixtures.html#conftest-py-sharing-fixtures-across-multiple-files)

Reply:
It doesn't work. Problem is that I am doing unittest mocking of `DatabankLib` module. And that didn't work if I do one mocking in one file and other mocking in the other and then run test for two files. It applied mocking only once. I put the clarification about it into heading of `test2_api.py`, and put TODO tag alongside.

Did that in commit 8ecf395c